src/Items/ItemBow.h

```diff
@@ -91,16 +91,15 @@ class cItemBowHandler :
 			}
 			else
 			{
-				Arrow->SetPickupState(cArrowEntity::psNoPickup);
+				ArrowPtr->SetPickupState(cArrowEntity::psNoPickup);
```

Comment:
I tested clang-tidy and found out that it's only checks the code in the source files, not in the header files. After creating `ItemBow.cpp` I got the use-after-move warning.